modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/processors/hdbschema/HDBSchemaCreateProcessor.java

```diff
@@ -29,13 +29,13 @@ public HDBSchemaCreateProcessor() {
   }
 
   public void execute(Connection connection, XSKDataStructureHDBSchemaModel hdbSchema) throws SQLException {
-    logger.info("Processing Create Schema: " + hdbSchema.getName());
+    logger.info("Processing Create Schema: " + hdbSchema.getSchema());
 
     ISqlDialect dialect = SqlFactory.deriveDialect(connection);
     if (!(dialect.getClass().equals(HanaSqlDialect.class))) {
       throw new IllegalStateException(String.format("%s does not support Schema", dialect.getDatabaseName(connection)));
     } else {
-      String sql = "CREATE SCHEMA " + hdbSchema.getName();
+      String sql = "CREATE SCHEMA " + hdbSchema.getSchema();
```

Comment:
Use escapeArtifactName method in order to take into consideration if we have caseSensitive flag set as true.
` String schemaName = XSKHDBUtils.escapeArtifactName(connection, hdbSchema.getSchema());`